examples/pointcloud_globe.js

```diff
@@ -0,0 +1,93 @@
+/* global itowns, debug, dat */
+var promises = [];
+// eslint-disable-next-line no-unused-vars
+function showPointcloud(serverUrl, fileName, lopocsTable) {
+    var pointcloud;
+    var oldPostUpdate;
+    var viewerDiv;
+    var debugGui;
+    var view;
+    var controls;
+
+    viewerDiv = document.getElementById('viewerDiv');
+    viewerDiv.style.display = 'block';
+
+    itowns.THREE.Object3D.DefaultUp.set(0, 0, 1);
+
+    itowns.proj4.defs('EPSG:3946',
+        '+proj=lcc +lat_1=45.25 +lat_2=46.75 +lat_0=46 +lon_0=3 +x_0=1700000 ' +
+        '+y_0=5200000 +ellps=GRS80 +towgs84=0,0,0,0,0,0,0 +units=m +no_defs');
+
+    debugGui = new dat.GUI();
+    
+    var positionOnGlobe = { longitude: 2.351323, latitude: 48.856712, altitude: 25000000 };
```

Comment:
You can use Arles precise coordinates
`var positionOnGlobe = { longitude: 4.631512, latitude: 43.675626, altitude: 250 };`